content/en/docs/doc-contributor-tools/linkchecker/README.md

```diff
@@ -0,0 +1,66 @@
+# Internal link checking tool
+
+You can use the https://github.com/wjdp/htmltest tool to check for broken links on **english content**. This is a helpful when refactoring sections of content, moving pages around, or renaming files or page headers.
+
+## What is and isn't checked
+
+The link checker scans generated HTML files, **not** raw markdown. You can see the full configuration in the `.htmltest.yml` file in the root of `kubernetes/website`.
+
+The link checker scans the following:
+
+- All content generated from markdown in the `/content/en/docs` directory **excluding**:
```

Comment:
Capitalize, link, omit needless words, remove emphasis
```suggestion
- All content generated from Markdown in [`/content/en/docs`](https://git.k8s.io/website/content/en/docs/) directory, excluding:
```

Reply:
TIL that `git.k8s.io` is a thing